libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -5961,6 +6043,18 @@ bool GCS_MAVLINK::send_relay_status() const
 }
 #endif  // AP_MAVLINK_MSG_RELAY_STATUS_ENABLED
 
+void GCS_MAVLINK::send_control_status() const
```

Comment:
Everything needs to be behind a feature define

Reply:
Noted. would MAVLINK_MULTIGCS_PROTOCOL_ENABLED work?